app/build.gradle

```diff
@@ -41,16 +41,23 @@ android {
 
     flavorDimensions "default"
     productFlavors {
+
         flavorAtest {
             applicationId "net.gsantner.markor_test"
             versionCode = Integer.parseInt(new Date().format('yyMMdd'))
             versionName = new Date().format('yyMMdd')
             buildConfigField "boolean", "IS_TEST_BUILD", "true"
+
+            vectorDrawables.useSupportLibrary = true
```

Comment:
We should be able to strike this one, or at least I would prefer to remove it. I made bad experience with `vectorDrawables.useSupportLibrary = true` in the past.